aws/cloudformation/cloud_formation_stack.yml.erb

```
@@ -401,6 +401,45 @@ Resources:
 
 <%   if cdn_enabled && app == 'Pegasus' -%>
 <%=    component 'www_redirect', app: app -%>
+  MarketingRouterLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Description: 'Lambda for dynamically routing marketing traffic to different origins based on the URL path'
+      FunctionName: <%="#{rack_env.to_s}-MarketingRouter"%>%
```

Does this work? I see a lot of `rack_env?` checks but no calls to `rack_env`, which appears to return a symbol representing the environment name.

---

Lets use stack name instead, and lower-kebab-case

```suggestion
      FunctionName: !Sub ${AWS::StackName}-marketing-router
```